tetrateci/fetch_all_tags.sh

```diff
@@ -29,12 +29,23 @@ for tag in $tags; do
     if [[ ! $(git rev-parse --verify --quiet origin/tetrate-release-$branch) ]]; then
         # create the tetrate release branch if it doesn't exist with the workflows
         git checkout -b tetrate-release-$branch origin/tetrate-workflow
+        git merge $tag --no-edit --allow-unrelated-histories
+        git tag test-$tag-tetrate-v0
+
+        git checkout -b tetratefips-release-$branch origin/tetrate-workflow
```

Comment:
Can we put fips in a separate if else block? Something similar to line 29 and 38 for non fips.

Reply:
Ok